network-api/networkapi/settings.py

```diff
@@ -462,30 +462,42 @@
 # LOGGING
 LOGGING = {
     'version': 1,
-    'disable_existing_loggers': False,
+    'disable_existing_loggers': True,
+    'filters': {
+        'require_debug_true': {
+            '()': 'django.utils.log.RequireDebugTrue',
+        }
+    },
     'formatters': {
-        'simple': {
-            'format': '[%(asctime)s] [%(levelname)s] %(message)s'
-        },
+        'verbose': {
+            'format': '%(asctime)s [%(levelname)s] %(message)s',
+            'datefmt': '%Y-%m-%d %H:%M:%S'
```

Comment:
do we want ms just in case we need to debug timing issues?

Reply:
These logs will only be present in dev. in Production we'll see logs like this: 
```
2019-01-16T19:14:45.762692+00:00 heroku[router]: at=info method=GET path="/_images/buyers-guide/gradient-bar.svg" host=foundation-mofostaging-pr-2400.herokuapp.com request_id=aebc9f38-b22c-47da-bdad-f4a7e76ef813 fwd="1.1.1.1" dyno=web.1 connect=0ms service=8ms status=200 bytes=1294 protocol=https
```

Reply:
...but yeah lets get more detailed timing info